kura/org.eclipse.kura.camel.cloud.factory/OSGI-INF/metatype/org.eclipse.kura.camel.cloud.factory.CamelFactory.xml

```diff
@@ -23,6 +23,13 @@
             cardinality="1"
             required="true"
             description="The camel XML router configuration"/>
+            
+         <AD id="serviceRanking"
+            name="serviceRanking"
+            type="Integer"
+            cardinality="0"
+            required="false"
```

Comment:
``serviceRanking`` seems to be ``required``. If left blank you get an exception in the Web UI.